src/cli/commands/block/get.js

```diff
@@ -17,7 +17,11 @@ module.exports = {
         throw err
       }
 
-      process.stdout.write(block.data)
+      if (block) {
+        process.stdout.write(block.data)
+      } else {
+        process.stderr.write('Block was unwanted before it could be remotely retrieved')
+      }
```

Comment:
Ideally we'd use `print` from utils so that output can be disabled (I'm not sure why you'd want to do that for this call but we should probably support it!).